src/commands/account/profile_management/update_profile/profile_args_type/mod.rs

```diff
@@ -0,0 +1,41 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod base64_args;
+mod file_args;
+mod json_args;
+mod manually;
+mod text_args;
+
+#[derive(Debug, EnumDiscriminants, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = super::UpdateAccountProfileContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// How do you want to pass profile arguments?
+pub enum ProfileArgsType {
+    #[strum_discriminants(strum(
+        message = "json-args    - Valid JSON arguments (e.g. {\"token_id\": \"42\"})"
+    ))]
+    /// Valid JSON arguments (e.g. {"token_id": "42"})
+    JsonArgs(self::json_args::JsonArgs),
+    #[strum_discriminants(strum(message = "text-args    - Arbitrary text arguments"))]
+    /// Arbitrary text arguments
+    TextArgs(self::text_args::TextArgs),
+    #[strum_discriminants(strum(message = "base64-args  - Base64-encoded string (e.g. e30=)"))]
+    /// Base64-encoded string (e.g. e30=)
+    Base64Args(self::base64_args::Base64Args),
+    #[strum_discriminants(strum(
+        message = "file-args    - Read from file (e.g. reusable JSON or binary data)"
+    ))]
+    /// Read from file (e.g. reusable JSON or binary data)
+    FileArgs(self::file_args::FileArgs),
+    #[strum_discriminants(strum(message = "manually     - Interactive input of arguments"))]
+    /// Interactive input of arguments
+    Manually(self::manually::Manually),
```

Comment:
Let's make "manually" the first option on the list since if someone uses interactive mode, it is probably the most desired option.

Reply:
Done